Examples/Davis/DavisData.py

```diff
@@ -29,6 +29,6 @@
 print("Solving Problem")
 fde.SolveProblem(.11)
 fde.Plot()
-
```

Comment:
adding the plot.show() here doesn't match the documentation. This is intentional in case users want to add other features (change labels, etc) to their plots.